app/src/main/java/com/jazart/symphony/MusicControls.java

```diff
@@ -0,0 +1,105 @@
+package com.jazart.symphony;
+
+/**
+ * Created by kendrickgholston on 7/9/18.
+ */
+import android.os.Handler;
+import android.widget.SeekBar;
+import android.widget.TextView;
+
+import com.google.android.exoplayer2.SimpleExoPlayer;
+
+import java.util.Formatter;
+import java.util.Locale;
+
+import static com.jazart.symphony.featured.MusicAdapter.exoPlayer;
+import static com.jazart.symphony.MainActivity.songPlaying;
+
+public class MusicControls {
+    public TextView txtCurrentTime, txtEndTime;
+    public static SeekBar playerSeek;
+    private Handler handler;
+    public MainActivity mA;
+    public MusicControls(MainActivity musicActive){
```

Comment:
We don't want to pass in an activity as an argument. This would lead to a memory leak.